iocore/net/P_QUICNetVConnection_native.h

```diff
@@ -0,0 +1,402 @@
+  Event *_packet_write_ready = nullptr;
+
+  void _schedule_closing_timeout(ink_hrtime interval);
+  void _unschedule_closing_timeout();
+  void _close_closing_timeout(Event *data);
+  Event *_closing_timeout = nullptr;
+
+  void _schedule_closed_event();
+  void _unschedule_closed_event();
+  void _close_closed_event(Event *data);
+  Event *_closed_event = nullptr;
+
+  void _schedule_ack_manager_periodic(ink_hrtime interval);
+  void _unschedule_ack_manager_periodic();
+  Event *_ack_manager_periodic = nullptr;
+
+  QUICEncryptionLevel _minimum_encryption_level = QUICEncryptionLevel::INITIAL;
+
+  QUICPacketNumber _largest_acked_packet_number(QUICEncryptionLevel level) const;
+  uint32_t _maximum_quic_packet_size() const;
+  uint32_t _minimum_quic_packet_size();
+  uint64_t _maximum_stream_frame_data_size();
+
+  Ptr<IOBufferBlock> _store_frame(Ptr<IOBufferBlock> parent_block, size_t &size_added, uint64_t &max_frame_size, QUICFrame &frame,
+                                  std::vector<QUICSentPacketInfo::FrameInfo> &frames);
+  QUICPacketUPtr _packetize_frames(uint8_t *packet_buf, QUICEncryptionLevel level, uint64_t max_packet_size,
+                                   std::vector<QUICSentPacketInfo::FrameInfo> &frames);
+  void _packetize_closing_frame();
+  QUICPacketUPtr _build_packet(uint8_t *packet_buf, QUICEncryptionLevel level, const Ptr<IOBufferBlock> &parent_block,
+                               bool retransmittable, bool probing, bool crypto);
+
+  QUICConnectionErrorUPtr _recv_and_ack(const QUICPacketR &packet, bool *has_non_probing_frame = nullptr);
+
+  QUICConnectionErrorUPtr _state_handshake_process_packet(const QUICPacket &packet);
+  QUICConnectionErrorUPtr _state_handshake_process_version_negotiation_packet(const QUICVersionNegotiationPacketR &packet);
+  QUICConnectionErrorUPtr _state_handshake_process_initial_packet(const QUICInitialPacketR &packet);
+  QUICConnectionErrorUPtr _state_handshake_process_retry_packet(const QUICRetryPacketR &packet);
+  QUICConnectionErrorUPtr _state_handshake_process_handshake_packet(const QUICHandshakePacketR &packet);
+  QUICConnectionErrorUPtr _state_handshake_process_zero_rtt_protected_packet(const QUICZeroRttPacketR &packet);
+  QUICConnectionErrorUPtr _state_connection_established_receive_packet();
+  QUICConnectionErrorUPtr _state_connection_established_process_protected_packet(const QUICShortHeaderPacketR &packet);
+  QUICConnectionErrorUPtr _state_connection_established_migrate_connection(const QUICPacketR &packet);
+  QUICConnectionErrorUPtr _state_connection_established_initiate_connection_migration();
+  QUICConnectionErrorUPtr _state_closing_receive_packet();
+  QUICConnectionErrorUPtr _state_draining_receive_packet();
+  QUICConnectionErrorUPtr _state_common_send_packet();
+  QUICConnectionErrorUPtr _state_handshake_send_retry_packet();
+  QUICConnectionErrorUPtr _state_closing_send_packet();
+
+  Ptr<ProxyMutex> _packet_transmitter_mutex;
```

Comment:
Doesn't look like this is used anywhere.

Reply:
Good catch. It has been on master (existing code) as well. I could make another PR to remove it on master but I just removed it on this branch to avoid merge conflicts.